client/package.json

```diff
@@ -39,20 +40,23 @@
     "@types/react-router-dom": "^5.1.5",
     "@types/urijs": "^1.19.9",
     "@types/uuid": "^8.0.0",
-    "@typescript-eslint/eslint-plugin": "^3.5.0",
+    "@typescript-eslint/eslint-plugin": "^2.34.0",
```

Comment:
Are we downgrading the ESLint plugin version?

Reply:
Put it back. idr why i did this. probably a dependency thing.